src/monitoring.c

```diff
@@ -208,6 +209,9 @@ static void handle_client(struct monitoring *monitoring, int fd)
 					json_object_new_boolean(monitoring->ctrl_values.lock));
 				json_object_object_add(oscillator, "temperature",
 					json_object_new_double(monitoring->temperature));
+				if (monitoring->phase_error_supported)
+					json_object_object_add(oscillator, "phase_error",
```

Comment:
I think we agreed on:
`ClockAccuracy`
`ClockClass`
Speaking of which -  I don't see the ClockClass implemented?

Reply:
I'm waiting for Charles to implement on their side and update our implementation then - don't actually want to mix code from different sources

Reply:
@vvfedorenko is phase error absolute value or can be negative?

Reply:
It could be negative according to SA5x user manual